fontc/src/args.rs

```diff
@@ -112,14 +118,22 @@ impl Args {
 mod tests {
     use clap::Parser;
     use fontir::orchestration::Flags;
+    use log::{debug, LevelFilter};
 
     use crate::Args;
 
     // It's awkward to get the Flags::default values into #[arg] so test for consistency
     #[test]
     fn arg_default_matches_flags_default() {
+        let _ = env_logger::builder()
+            .is_test(true)
+            .format_timestamp(None)
+            .filter_level(LevelFilter::Debug)
+            .try_init();
         let arg_default = Args::parse_from(vec!["program", "--source", "dont.care"]).flags();
         let flags_default = Flags::default();
+        debug!("flags_default: {:#032b}", flags_default.bits());
+        debug!("args_default:  {:#032b}", arg_default.bits());
```

Comment:
Instead of firing up the logger and writing to debug make the assert do what you want, either by altering the left/right values or by adding a custom panic string to show the `{:#032b}` strings